src/main/java/core/basesyntax/FigurePrinterInfo.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigurePrinterInfo {
```

Comment:
**FigurePrinterInfo** is not **abstract** name. The interface should be independent. Do not forget Abstraction OOP principle.
```suggestion
public interface Drawable {
```